test/e2e/gitops_service_test.go

```diff
@@ -278,6 +278,13 @@ func tearDownArgoCD(t *testing.T) {
 }
 
 func validateMachineConfigUpdates(t *testing.T) {
+
+	// This test will fail automation until gitops-operator #148 is fixed.
+	// 'When GitOps operator is run locally (not installed via OLM), it does not correctly setup
+	// the 'argoproj.io' Role rules for the 'argocd-application-controller'
+	// (https://github.com/redhat-developer/gitops-operator/issues/148)
+	t.SkipNow()
```

Comment:
@jgwest We have a test target https://github.com/redhat-developer/gitops-operator/blob/master/Makefile#L31 that runs the same test on gitops-operator installed via OLM. We can skip this test with a condition, for example
```
if os.Getenv("SKIP_OPERATOR_DEPLOYMENT") != "true" {
		t.SkipNow()
	}
```

Reply:
Done.